src/Congo.WebApi.Tests/Controllers/CartControllerTests.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using Congo.WebApi.Controllers;
+using FluentAssertions;
+using MediatR;
+using Microsoft.AspNetCore.Mvc;
+using Moq;
+using Xunit;
+
+namespace Congo.WebApi.Tests
+{
+    public class CartControllerTests
+    {
+        private readonly Mock<IMediator> mediatorMoq;
+        private CartController _subject;
```

Comment:
In your other tests, this is readonly. It should be that here too :) 
```suggestion
        private readonly CartController _subject;
```

Reply:
Fixed